### PR TITLE
DBZ-1379 Fix PostgreSQL false empty password Warn

### PR DESCRIPTION
**Pull request by renatomefi**
Ticket: https://issues.jboss.org/browse/DBZ-1379
Inspired by: https://github.com/debezium/debezium/pull/816

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java [hunk 1]
```diff
@@ -84,8 +86,9 @@ public Config validate(Map<String, String> connectorConfigs) {
         ConfigValue databaseValue = results.get(PostgresConnectorConfig.DATABASE_NAME.name());
         ConfigValue userValue = results.get(PostgresConnectorConfig.USER.name());
         ConfigValue passwordValue = results.get(PostgresConnectorConfig.PASSWORD.name());
+        final String passwordStringValue = config.getConfig().getString(PostgresConnectorConfig.PASSWORD);
 
-        if ( passwordValue.value() == null || ((String) passwordValue.value()).isEmpty() ) {
+        if (Strings.isNullOrEmpty(passwordStringValue)) {
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java]: So this fixes the issue, but isn't there some deeper issue to this? Why is it that `passwordValue.value()` returns null? That seems not right.

Comment (renatomefi) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java]: Yes I agree, I must be honest I didn't dive into it, I think to reproduce it is quite tricky, it seems to happen when the connector restarts or so

Comment (gunnarmorling) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java]: Ok, so it seems the values of `ConfigValue` are always null in our case. The usage of the KC config framework seems a bit odd, but should be fine for now. Merging.

**Comment by gunnarmorling** [comment]
Applied; thanks, @renatomefi!

**Pull request merged**